client/home/lib/virtualmachines/components/machineslist/machinedetails.coffee

```diff
@@ -217,16 +218,17 @@ generateSpecs = (machine) ->
 
   specs = []
 
+  providerName = jMachine.meta?.type
+  instanceType = jMachine.meta?.instance_type ? 't2.micro'
   size = jMachine.meta?.storage_size
 
+  { ram } = providerSpecs[providerName][instanceType]
+  { cpu } = providerSpecs[providerName][instanceType]
```

Comment:
@rjeczalik Can you please direct us to the correct place where we can find the implemented `provider`s and their specs? Especially i am interested in to see what the defaults are for each provider, so that we can have that info and show those to the users correctly.